testsuite/cli/libra-wallet/src/mnemonic.rs

```diff
@@ -77,7 +77,9 @@ impl Mnemonic {
         bit_writer.write_buffer();
 
         // This will never fail as we've already checked the word-list is not empty.
-        let (checksum, entropy) = bit_writer.bytes.split_last().unwrap();
+        let (checksum, entropy) = bit_writer.bytes.split_last().ok_or_else(|| {
+            WalletError::LibraWalletGeneric("Bit writer producing inconsistent byte strings".into())
```

Comment:
what happens if we hit this? is it really safe to continue on in the libra-wallet? how do we handle this upstream?